src/shared/components/table/table.css

```diff
@@ -5,17 +5,30 @@
     border:none;
     width:100%;
     text-align: left;
+    table-layout: fixed;
+  }
+    
+  &__column th, &__column td {
```

Comment:
Let's make these --column instead
Considering BEM, this would not be an element of the hw-table but a modifier to it.